client/src/pages/Profile/Profile.tsx

```diff
@@ -0,0 +1,56 @@
+import { useState } from "react"
+import { Grid, Container, Typography, Link } from '@material-ui/core';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import useStyles from './useStyles';
+import EditProfile from '../../components/EditProfile/EditProfile'
+import ProfilePhoto from '../../components/ProfilePhoto/ProfilePhoto'
+import Payment from '../../components/Payment/Payment'
+import Security from '../../components/Security/Security'
+import Settings from '../../components/Settings/Settings'
+
+
+export default function Profile(props: any): JSX.Element {
+
```

Comment:
In TypeScript we can define the type the props is expected to receive by creating an interface and doing `Profile(props: ProfileProps)`